iam/README.md

```diff
@@ -43,20 +43,23 @@ will be processed independent of others.
-| `name`                     | The name of the service system                                                                   | No      |
-| `permission-prefix`        | The permission prefix, this will be prefixed to every permission and roles id. ([a-z][-a-z]{2})  | Yes      |
-| `services`                 | A list of services that need a DAS system                                                        | Yes      |
-| `services.name`            | The service name of the system to be created                                                     | Yes      |
-| `services.repository`      | The description of the service                                                                   | Yes      |
-| `permissions`              | An object containing the permissions for this service. (keys [a-z][-a-z]{1,15})                  | Yes      |
-| `prmissions.<resource>`    | Array containing the verbs for the `<resource>`. (items [a-z][-a-z]{1,15})                       | Yes      |
-| `roles`                    | An list of roles that should exist for this service                                              | No       |
-| `roles.id`                 | The role id ([a-z][-a-z]{1,19})                                                                  | Yes      |
-| `roles.name`               | The role name                                                                                    | Yes      |
-| `roles.desc`               | The description of the role, (max 200 characters)                                                | Yes      |
-| `roles.permissions`        | An list of permissions this role should contain (items ^[a-z][-a-z]{1,15}\.[a-z][-a-z]{1,15}$)   |  Yes      |
+| Property                    | Description                                                                                      | Required |
+|:----------------------------|:-------------------------------------------------------------------------------------------------|:---------|
+| `name`                      | The name of the service system                                                                   | No      |
+| `permission-prefix`         | The permission prefix, this will be prefixed to every permission and roles id. ([a-z][-a-z]{2})  | Yes      |
+| `services`                  | A list of services that need a DAS system                                                        | Yes      |
+| `services.name`             | The service name of the system to be created                                                     | Yes      |
+| `services.repository`       | The description of the service                                                                   | Yes      |
+| `services.allowed-consumers`| A list of clans and services allowed to consume this service                                     | No       |
+| `services.clan`             | Your clan name subscribing to this service                                                       | Yes      |
```

Comment:
I think this key needs to be `services.allowed-consumers.clan`. This way to annotate is probably not correct. Maybe it should be something like `services[].allowed-consumers[].clan`, but I guess the first form would clear enough.

Reply:
Fixed, full path to property added